dojo/management/commands/initialize_test_types.py

```diff
@@ -1,9 +1,14 @@
 from django.core.management.base import BaseCommand
-from dojo.tools.factory import initialize_test_types
+from dojo.tools.factory import PARSERS
+from dojo.models import Test_Type
 
 
 class Command(BaseCommand):
     help = 'Initializes Test_Types'
 
     def handle(self, *args, **options):
-        initialize_test_types()
+        # called by the initializer to fill the table with test_types
+        for scan_type in PARSERS:
+            test_type, created = Test_Type.objects.get_or_create(name=scan_type)
+            if created:
+                test_type.save()
```

Comment:
Saving is not necessary with `get_or_create`

Reply:
I just moved the code but I can refactor it this way.

Reply:
Fixed